charts/falco/values-k8saudit.yaml

```diff
@@ -18,22 +18,20 @@ controller:
 falcoctl:
   artifact:
     install:
-      # -- Enable the init container. We do not recommend installing (or following) plugins for security reasons since they are executable objects.
+      # -- Enable the init container.
       enabled: true
     follow:
-      # -- Enable the sidecar container. We do not support it yet for plugins. It is used only for rules feed such as k8saudit-rules rules.
+      # -- Enable the sidecar container.
       enabled: true
   config:
     artifact:
       install:
-        # -- Do not resolve the depenencies for artifacts. By default is true, but for our use case we disable it.
-        resolveDeps: false
+        # -- Resolve the dependencies for artifacts.
+        resolveDeps: true
         # -- List of artifacts to be installed by the falcoctl init container.
-        # Only rulesfiles, we do no recommend plugins for security reasonts since they are executable objects.
         refs: [k8saudit-rules:0.6]
```

Comment:
Note: we are going to release `k8saudit-rules:0.7`, so this will need to be updated.

Reply:
@alacuku https://github.com/falcosecurity/plugins/releases/tag/k8saudit-0.7.0 is out (and also `k8saudit-rules:0.7`)